gomint-server/src/main/java/io/gomint/server/entity/EntityPlayer.java

```diff
@@ -745,21 +742,21 @@ public void prepareEntity() {
 
         this.windowIds = new Byte2ObjectOpenHashMap<>();
         this.containerIds = new Object2ByteOpenHashMap<>();
-        this.connection.getServer().getCreativeInventory().addViewer(this);
+        // TODO: this.connection.getServer().getCreativeInventory().addViewer(this);
```

Comment:
I commented it because the data it sends it's outdated and wihout MITM we cannot update it at the moment, i will uncomment as soon as we dump updated data, always if for you is ok (this for all the other comment you see)